Documentation/network.md

```diff
@@ -56,3 +55,4 @@ dns=8.8.8.8;8.8.4.4;
 ```
 
 [keyfile]: https://developer.gnome.org/NetworkManager/stable/nm-settings.html
+[configuration-usb]: https://github.com/home-assistant/hassos/blob/rel-1/Documentation/configuration.md
```

Comment:
We can't linking on a branch. Do you know how we can make a link inside branch?

Reply:
Just use relative path if it's same repository [configuration-usb](configuration.md)